code/game/turfs/walls/wall_types.dm

```diff
@@ -17,6 +17,7 @@
 	///do we have bits of decoration to add to the walls?
 	var/decorated_wall = TRUE
 
+/* this completely breaks hull walls in HvH, will fix in a separate pr because wall overlays are horribly shitcode
 /turf/closed/wall/mainship/smooth_icon()
```

Comment:
????????

Reply:
Yeah this code was scuffed, and was causing unplayable bugs for hvh so I couldn't test without disabling it. 

I can just remove the code entirely/revert the pr that caused it, because a proper fix is needed in another pr (wall icon code is super wack in general though).